revenuecat_examples/purchase_tester/lib/src/upsell.dart

```diff
@@ -170,9 +171,39 @@ class _UpsellScreenState extends State<UpsellScreen> {
                 padding: const EdgeInsets.all(8.0),
                 child: Column(children: [
                   const Text("Customer Center"),
+                  ElevatedButton(
+                    onPressed: () {
+                      Navigator.push(
+                        context,
+                        MaterialPageRoute(
+                          builder: (_) => const CustomerCenterViewModalScreen(),
+                        ),
+                      );
+                    },
+                    child: const Text("CustomerCenterViewModalScreen (Close Button)"),
+                  ),
+                  const SizedBox(height: 12),
                   ElevatedButton(
                     onPressed: () async {
-                      await RevenueCatUI.presentCustomerCenter();
+                      await RevenueCatUI.presentCustomerCenter(
+                        onRestoreStarted: () => debugPrint('[CustomerCenter Modal] Restore started'),
+                        onRestoreCompleted: (customerInfo) =>
+                            debugPrint('[CustomerCenter Modal] Restore completed: ${customerInfo.originalAppUserId}'),
+                        onRestoreFailed: (error) =>
+                            debugPrint('[CustomerCenter Modal] Restore failed: ${error.message}'),
+                        onShowingManageSubscriptions: () =>
+                            debugPrint('[CustomerCenter Modal] Showing manage subscriptions'),
+                        onRefundRequestStarted: (productId) =>
+                            debugPrint('[CustomerCenter Modal] Refund request started for product: $productId'),
+                        onRefundRequestCompleted: (productId, status) =>
+                            debugPrint('[CustomerCenter Modal] Refund request completed for product $productId with status $status'),
+                        onFeedbackSurveyCompleted: (optionId) =>
+                            debugPrint('[CustomerCenter Modal] Feedback survey completed with option: $optionId'),
+                        onManagementOptionSelected: (optionId, url) =>
+                            debugPrint('[CustomerCenter Modal] Management option selected: $optionId (url: ${url ?? 'none'})'),
+                        onCustomActionSelected: (actionId, purchaseIdentifier) =>
+                            debugPrint('[CustomerCenter Modal] Custom action selected: $actionId (purchase: ${purchaseIdentifier ?? 'none'})'),
+                      );
```

Comment:
super nit but maybe adding a "debugPrint('[CustomerCenter] Dismissed');" after the await would be nice to make sure that's behaving as expected :)